recipe/meta.yaml

```diff
@@ -18,11 +18,10 @@ requirements:
     - python >=3.5
     - pip
   run:
-    - python >=3.5
```

Comment:
Not sure, but I might think you need to keep the version specification here, as otherwise conda might think it can be installed on any python version

Reply:
Could you suggest code change for that please? Not sure how I'd go about it.

Reply:
Keep what was there?